core/ui/src/main/kotlin/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -112,7 +128,22 @@ fun NewsResourceCardExpanded(
                     Row {
                         NewsResourceTitle(
                             userNewsResource.title,
-                            modifier = Modifier.fillMaxWidth((.8f)),
+                            modifier = Modifier.fillMaxWidth((.8f))
+                                .dragAndDropSource {
+                                    detectTapGestures(
+                                        onLongPress = {
+                                            startTransfer(
+                                                DragAndDropTransferData(
+                                                    ClipData.newPlainText(
+                                                        "Feed sharing",
+                                                        userNewsResource.title + ": " + userNewsResource.url,
```

Comment:
Since this is the string manipulation that the user can see, should this be using string resources to perform the manipulation in case that would need to be translated?

Reply:
That would require Context. Would you be OK in passing it down from the Composable itself?